test/Examples/Stylish.hs

```diff
@@ -0,0 +1,60 @@
+module Devtools
```

Comment:
I've gotta say, Stylish looks a bit more like our "common style" than the others.

Also I like how it does not bother to align the `as` in imports, this will cause a lot of churn (although I do it when I am formatting by hand).

It also eliminates unnecessary whitespace from lots of places instead of leading it "just in case" -- although the other tools may be leaving the whitespace to minimize churn too.

Reply:
Note that this is pre-configuration. I suspect that the others can also be "bend" more to our style.

Other than that: I predict once formatting is automatic we do not "see" style that much anymore.

Reply:
Stylish style configuration scope is mainly around module imports so its very narrow. Brittany has more opinions than we can bend to our style.